sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlCliTest.java

```diff
@@ -48,6 +49,29 @@ public void testExecute_createTextTable() throws Exception {
     assertNotNull(table);
   }
 
+  @Test
+  public void testExecute_dropTable() throws Exception {
```

Comment:
can you add another unit test to see if the table is not available in `Planner` as expected?